kale/utils/download.py

```diff
@@ -0,0 +1,71 @@
+# =============================================================================
+# Author: Xianyuan Liu, xianyuan.liu@sheffield.ac.uk
```

Comment:
You may consider to start using your more permanent, personal email (or both emails, see my case at https://github.com/pykale/pykale/blob/main/kale/embed/mpca.py) as your Shef email will become invalid soon after you leave.